onnxruntime_extensions/cvt.py

```diff
@@ -47,6 +204,9 @@ def gen_processing_models(processor: Union[str, object],
         Keyword arguments for generating the post-processing model
     opset: int
         the target opset version of the model
+    cached: bool
```

Comment:
can a better name for this argument? 'cached' look like being irrelevant to what we are doing here.